services/QuillLMS/spec/models/cron_spec.rb

```diff
@@ -121,6 +121,7 @@
     end
 
     it "enqueues TeacherNotifications::EnqueueUsersForRollupEmailWorker" do
+      expect(TeacherNotifications::EnqueueUsersForRollupEmailWorker).to receive(:perform_async).with(TeacherInfo::WEEKLY_EMAIL) if Time.zone.today.wday == 5
```

Comment:
Is this supposed to be in here?

Reply:
Removed! Was fixing a broken spec I noticed on develop but it just got fixed in another PR, so this is unneeded.